### PR TITLE
Fix Spotlights Summer 2020 leaderboard

### LINKED ISSUE #4386: Beatmap Spotlights Summer 2020 leaderboard incorrect
The leaderboard shown on the wiki seems to be an outdated one which is missing some players and misplaces others, and displays the badge assets one tier lower than they are in reality. Although image below is from standard this does occur on all mode leaderboards.

Untitled

### PR DESCRIPTION
**Pull request by Shadowthegod**
fixes #4386

### CODE REVIEW

File: wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md [hunk 1]
```diff
@@ -88,49 +88,49 @@ The following leaderboards are also available as a [Google Spreadsheet](https://
...
+| #29 | [King Hong](https://osu.ppy.sh/users/7263047) | ![][osu-diamond1] |
+| #30 | [Shigeru22](https://osu.ppy.sh/users/2581664) | ![][osu-diamond1] |
+| #31 | [enjoyit1025](https://osu.ppy.sh/users/2206844) | ![][osu-diamond1] |
+| #32 | [TableRaw](https://osu.ppy.sh/users/4372688) | ![][osu-platinum2] |
+| #33 | [Lexion](https://osu.ppy.sh/users/5271371) | ![][osu-platinum2] |
+| #34 | [shdewz](https://osu.ppy.sh/users/10000899) | ![][osu-platinum2] |
+| #35 | [Nyakko](https://osu.ppy.sh/users/4383554) | ![][osu-platinum2] |
+| #36 | [Kahli](https://osu.ppy.sh/users/8926244) | ![][osu-platinum2] |
+| #37 | [Buszek](https://osu.ppy.sh/users/6867478) | ![][osu-platinum2] |
+| #38 | [Bitcoin](https://osu.ppy.sh/users/654296) | ![][osu-platinum2] |
```
[the remark this answers is not in the document]
Comment (Shadowthegod) [reply wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md]: fixed

File: wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md [hunk 2]
```diff
@@ -206,49 +206,49 @@ The following leaderboards are also available as a [Google Spreadsheet](https://
...
+| #24 | [Titan-](https://osu.ppy.sh/users/7855767) | ![][catch-gold2] |
+| #25 | [Novoids](https://osu.ppy.sh/users/5163523) | ![][catch-gold2] |
+| #26 | [\[\_\-Kukkai\-\_\]](https://osu.ppy.sh/users/7811952) | ![][catch-gold2] |
+| #27 | [Gordon123](https://osu.ppy.sh/users/2789905) | ![][catch-gold2] |
+| #28 | [by-ad](https://osu.ppy.sh/users/7439939) | ![][catch-gold2] |
+| #29 | [Roido](https://osu.ppy.sh/users/6829103) | ![][catch-gold2] |
+| #30 | [Spectator](https://osu.ppy.sh/users/702598) | ![][catch-gold2] |
+| #31 | [Kasumii-sama](https://osu.ppy.sh/users/6177263) | ![][catch-gold2] |
+| #32 | [Mafumafu](https://osu.ppy.sh/users/3076909) | ![][catch-gold2] |
+| #33 | [\-Joni\-](https://osu.ppy.sh/users/9988837) | ![][catch-gold2] |
```
Comment (TPGPL) [inline wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md:226]:
```suggestion
| #33 | [-Joni-](https://osu.ppy.sh/users/9988837) | ![][catch-gold2] |
```

File: wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md [hunk 3]
```diff
@@ -206,49 +206,49 @@ The following leaderboards are also available as a [Google Spreadsheet](https://
...
-| #47 | [PokeSky](https://osu.ppy.sh/users/3617111) | ![][catch-gold1] |
-| #48 | [K 3 V R A L](https://osu.ppy.sh/users/9852829) | ![][catch-gold1] |
-| #49 | [Baron](https://osu.ppy.sh/users/10286499) | ![][catch-gold1] |
-| #50 | [Faputa](https://osu.ppy.sh/users/845733) | ![][catch-gold1] |
+| #21 | [Netamaru](https://osu.ppy.sh/users/1830361) | ![][catch-platinum1] |
+| #22 | [vexAkita](https://osu.ppy.sh/users/8923804) | ![][catch-platinum1] |
+| #23 | [enjoyit1025](https://osu.ppy.sh/users/2206844) | ![][catch-platinum1] |
+| #24 | [Titan-](https://osu.ppy.sh/users/7855767) | ![][catch-gold2] |
+| #25 | [Novoids](https://osu.ppy.sh/users/5163523) | ![][catch-gold2] |
+| #26 | [\[\_\-Kukkai\-\_\]](https://osu.ppy.sh/users/7811952) | ![][catch-gold2] |
```
Comment (TPGPL) [inline wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md:219]:
```suggestion
| #26 | [\[\_-Kukkai-\_\]](https://osu.ppy.sh/users/7811952) | ![][catch-gold2] |
```

File: wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md [hunk 4]
```diff
@@ -206,49 +206,49 @@ The following leaderboards are also available as a [Google Spreadsheet](https://
...
+| #41 | [aiyulu](https://osu.ppy.sh/users/189617) | ![][catch-gold1] |
+| #42 | [Kerilia](https://osu.ppy.sh/users/7990615) | ![][catch-gold1] |
+| #43 | [Hidden is fun](https://osu.ppy.sh/users/10449071) | ![][catch-gold1] |
+| #44 | [Zech](https://osu.ppy.sh/users/5022932) | ![][catch-gold1] |
+| #45 | [catch the Exgon](https://osu.ppy.sh/users/12081746) | ![][catch-gold1] |
+| #46 | [Dabo](https://osu.ppy.sh/users/9507660) | ![][catch-gold1] |
+| #47 | [Icymice](https://osu.ppy.sh/users/8205964) | ![][catch-gold1] |
+| #48 | [PokeSky](https://osu.ppy.sh/users/3617111) | ![][catch-gold1] |
+| #49 | [K 3 V R A L](https://osu.ppy.sh/users/9852829) | ![][catch-gold1] |
+| #50 | [Baron](https://osu.ppy.sh/users/10286499) | ![][catch-gold1] |
```
Comment (TPGPL) [inline wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md:243]: image

File: wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md [hunk 5]
```diff
@@ -261,41 +261,41 @@ The following leaderboards are also available as a [Google Spreadsheet](https://
...
-| #34 | [cocoalatte](https://osu.ppy.sh/users/9059651) | ![][mania-gold1] |
-| #35 | [Doofyplayer](https://osu.ppy.sh/users/9828552) | ![][mania-gold1] |
-| #36 | [Joylintp](https://osu.ppy.sh/users/13579078) | ![][mania-gold1] |
-| #37 | [Sebaex](https://osu.ppy.sh/users/4686036) | ![][mania-gold1] |
+| #16 | [enjoyit1025](https://osu.ppy.sh/users/2206844) | ![][mania-platinum1] |
+| #17 | [\[Illustrious\]](https://osu.ppy.sh/users/4082595) | ![][mania-platinum1] |
+| #18 | [Sebaex](https://osu.ppy.sh/users/4686036) | ![][mania-platinum1] |
+| #19 | [Hellnote102](https://osu.ppy.sh/users/2218066) | ![][mania-platinum1] |
+| #20 | [akace100](https://osu.ppy.sh/users/9308128) | ![][mania-platinum1] |
+| #21 | [tsusai](https://osu.ppy.sh/users/7637292) | ![][mania-platinum1] |
```
Comment (TPGPL) [inline wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md:269]: image

File: wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md [hunk 6]
```diff
@@ -261,41 +261,41 @@ The following leaderboards are also available as a [Google Spreadsheet](https://
...
+| #23 | [Hidden is fun](https://osu.ppy.sh/users/10449071) | ![][mania-gold2] |
+| #24 | [Muse Dash](https://osu.ppy.sh/users/13695676) | ![][mania-gold2] |
+| #25 | [medium kek](https://osu.ppy.sh/users/11625617) | ![][mania-gold2] |
+| #26 | [Suichan203](https://osu.ppy.sh/users/9821346) | ![][mania-gold2] |
+| #27 | [Cynplytholowazy](https://osu.ppy.sh/users/3901754) | ![][mania-gold2] |
+| #28 | [GhostRevenge](https://osu.ppy.sh/users/8980266) | ![][mania-gold2] |
+| #29 | [Crystal](https://osu.ppy.sh/users/1646397) | ![][mania-gold2] |
+| #30 | [Raveille](https://osu.ppy.sh/users/1388767) | ![][mania-gold2] |
+| #31 | [Gato2499](https://osu.ppy.sh/users/7857559) | ![][mania-gold2] |
+| #32 | [Dilly_P](https://osu.ppy.sh/users/12907431) | ![][mania-gold1] |
```
Comment (TPGPL) [inline wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md:280]:
```suggestion
| #32 | [Dilly\_P](https://osu.ppy.sh/users/12907431) | ![][mania-gold1] |
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md (hunk 5 above)
Comment (Shadowthegod) [reply wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md]: Eh, I missed this cause it doesn't really matter but fixed nevertheless.

File: wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md (hunk 4 above)
Comment (Shadowthegod) [reply wiki/Beatmap_Spotlights/Seasons/2020_Summer/en.md]: 👍

**Review by TPGPL — APPROVED** [review]

**Pull request merged**